shell/init_bag_module.sh

```diff
@@ -23,10 +23,10 @@ DESCRIPTION
    Run this from the Virtuoso directory. 
 
 OPTIONS
-  -t  
-       Target module name
-  -w 
-      Working directory. Default: current directory.
+  -t
```

Comment:
Removed the options that are not really used

Reply:
Leave the -w